Use Kurtosis to run relayer and validator docs (WO URL) (#10)
hyperlane-xyz/v3-docs: docs/deploy-hyperlane.mdx

:::

![Example Configuration One](../static/img/kurtosis-example-configuration-one.png)

![Example Configuration Two](../static/img/kurtosis-example-configuration-two.png)

Once the configuration finishes executing successfully, you'll see a green check mark. Congratulations! You've now deployed your own relayer and validator with Kurtosis! 

![Success](../static/img/kurtosis-success.png)

You can now see information about your relayer and validator deployment by navigating to `Overview`. You can also view the relayer and validators logs along with other information about the services (see screenshots below).

![Enclave Overview](../static/img/kurtosis-enclave-overview.png)

![Validator Logs](../static/img/kurtosis-validator-logs.png)

For feedback or questions about using Kurtosis in this section, please file a [Github Issue](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) or start a [Github Discussion](https://github.com/kurtosis-tech/kurtosis/discussions/new?category=q-a). Additionally, the Kurtosis team is available for technical support on their [Discord server](https://discord.com/invite/jJFG7XBqcY).

:::info

> Kurtosis Cloud is available for a one month free trial. To continue using Kurtosis for deploying your setup beyond the trial, you can provide your payment information [https://cloud.kurtosis.com/payment].

Author: @mieubrisse any recs on this wording?

Author: ```
Kurtosis Cloud is available for a one month free trial. To continue using Kurtosis beyond you're free trail, see: https://cloud.kurtosis.com/payment.
```

:::

![Example Configuration One](../static/img/kurtosis-example-config-one.png)

![Example Configuration Two](../static/img/kurtosis-example-config-two.png)

Click `Run` to deploy your relayer and validator!

Once the configuration finishes executing successfully, you'll see a green check mark. Congratulations! You've now deployed your own relayer and validator with Kurtosis! 

![Success](../static/img/kurtosis-success.png)

You can see information about your relayer and validator deployment, including logs, by navigating to the `Overview` tab.

![Enclave Overview](../static/img/kurtosis-enclave-overview.png)

![Validator Logs](../static/img/kurtosis-validator-logs.png)

For issues with Kurtosis, please file an [issue on the Kurtosis Github repository](https://github.com/kurtosis-tech/kurtosis/issues/new/choose). For questions or discussion, use the [Github Discussions section of the Kurtosis repository](https://github.com/kurtosis-tech/kurtosis/discussions/new?category=q-a) or visit [the Kurtosis Discord server](https://discord.com/invite/jJFG7XBqcY).

:::info

> Kurtosis offers a one-month free trial of Kurtosis Cloud for running your relayer and validator, but when your trial expires you'll need to add a payment method to continue running your Hyperlane deployment. Subscription prices and payment method information can be accessed [here](https://cloud.kurtosis.com/payment-method).

Reviewer: ```suggestion
Kurtosis offers a one-month free trial of Kurtosis Cloud for running your relayer and validator, but when your trial expires you'll be prompted to add a payment method to continue running your Hyperlane deployment. Kurtosis Cloud subscription information is available [here](https://www.kurtosis.com/pricing).
```
